logic/src/commonTest/kotlin/com/wire/kalium/logic/data/asset/AssetRepositoryTest.kt

```diff
@@ -252,6 +270,38 @@ class AssetRepositoryTest {
         }
     }
 
+    @Test
+    fun givenAnAssetId_whenDownloadingPrivateAssetWithWrongAssetHash_thenShouldReturnAWrongAssetHashError() = runTest {
```

Comment:
Please add the following comment above this test:
`// @SF.Messages @TSFI.UserInterface @S0.1 @S1`
This will later help to collect this test as part of the security test documentation